app/templates/base.html

```diff
@@ -31,44 +30,113 @@
     <div class="blue-overlay">
 
     </div>
-{% endblock %}
-
-{% block content %}
+    {% endblock %} {% block content %}
     <div class="container">
         {% for message in get_flashed_messages() %}
-            <div class="alert alert-success">
-                <button type="button" class="close" data-dismiss="alert">&times;</button>
-                {{ message }}
-            </div>
-        {% endfor %}
-
-        {% block page_content %}{% endblock %}
+        <div class="alert alert-success">
+            <button type="button" class="close" data-dismiss="alert">&times;</button>
+            {{ message }}
+        </div>
+        {% endfor %} {% block page_content %}{% endblock %}
     </div>
+        <div class="div-wrap">
     <div class="blue-footer">
-{% endblock %}
-
-{% block footer %}
+        <div class="center-footer">
+            <div class="col-sm-12">
+                <div class="footer-left col-sm-3">
+                    <div class="footer-box">
+                        <img class="doris-banner" src="/static/img/doris_banner.png">
```

Comment:
Instead of hardcoding this image path we want to use the flask and jinja method of `url_for` it would look like `{{ url_for('static', filename='img/doris_banner.png') }}`